plugins/module_utils/network/ios/argspec/bgp_address_family/bgp_address_family.py

```diff
@@ -163,6 +163,7 @@ class Bgp_AddressFamilyArgs(object):
                         "neighbor": {
                             "type": "list",
                             "elements": "dict",
+                            "aliases": ["neighbors"],
```

Comment:
We should be rendering as neighbors too in facts. It will probably not be a breaking change since both the words are accepted by the module.